docs/intro/arch_overview/outlier.rst

```diff
@@ -98,12 +103,39 @@ action
   brought back into service.
 
 type
-  If ``action`` is ``eject``, species the type of ejection that took place. Currently this can
-  only be ``5xx``.
+  If ``action`` is ``eject``, specifies the type of ejection that took place. Currently this can be:
+  ``5xx``, and ``SuccessRate``.
 
 num_ejections
-  The number of times the host has been ejected (local to that Envoy and gets reset if the host
-  gets removed from the upstream cluster for any reason and then re-added).
+  If ``action`` is ``eject``, specifies the number of times the host has been ejected
+  (local to that Envoy and gets reset if the host gets removed from the upstream cluster for any
+  reason and then re-added).
+
+enforced
+  If ``action`` is ``eject``, specifies if the ejection was actually performed (``true``), or
+  if the action was only logged but the host was not ejected (``false``).
+
+host_success_rate
+  If ``action`` is ``eject``, and ``type`` is ``SuccessRate``, specifies the host's success rate
+  at the time of the ejection event on a ``0-100`` range. A ``-1`` value means that a success
```

Comment:
-1 can only happen if type is 5xx, right? This is kind of confusing. Can we just not write out host_succes_rate in the case of 5xx? Same for all the ones below.